src/sphinx_py3doc_enhanced_theme/static/pydoctheme.css_t

```diff
@@ -284,13 +288,17 @@ li > img {
 }
 
 h1, h2, h3, h4, h5, h6 {
+    {% if theme_headshadow %}
     text-shadow: 1px 1px 2px #ddd;
+    {% endif %}
     font-weight: bold;
     color: #333;
 }
 code, tt {
     background: none;
+    {% if codeshadow %}
```

Comment:
Ah whoops. I also just found your CONTRIBUTING.rst... muh bad... I don't think I followed any of your recommendations.

Anyways, PR coming up.